pkgs/top-level/all-packages.nix

```diff
@@ -361,9 +361,11 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) Security;
   };
 
+  pridecat = callPackage ../tools/misc/pridecat { };
+
   proto-contrib = callPackage ../development/tools/proto-contrib { };
 
-  protoc-gen-doc = callPackage ../development/tools/protoc-gen-doc {};
+  protoc-gen-doc = callPackage ../development/tools/protoc-gen-doc { };
```

Comment:
Not really sure if this should be touched (even though it really doesn't matter), since it doesn't relate to this package. @Sciencentistguy thoughts?

Reply:
I think this is the preferred formatting, but it is related. I'd still merge with that change if it were up to me, I think.

Reply:
Oh, this isn't the same line that I commented on, its another one that got caught up. still, i'd probably leave it